src/components/TopRow.tsx

```diff
@@ -1,44 +1,90 @@
-import Link from "next/link";
-import { Box, HStack, Text, Icon } from "@chakra-ui/react";
-import { CustomConnectButton } from "components/CustomConnectButton";
+import { Box, HStack, Icon, Text } from "@chakra-ui/react";
 import { ColorModeSwitcher } from "components/ColorModeSwitcher";
-import { ImStatsBars as StatsIcon, ImTable as TableIcon } from "react-icons/im";
+import { CustomConnectButton } from "components/CustomConnectButton";
+import { useGlobalContext } from "contexts/GlobalContext";
+import Link from "next/link";
+import router, { useRouter } from "next/router";
+import { useEffect } from "react";
 import { CgCardSpades as CardIcon } from "react-icons/cg";
+import { ImStatsBars as StatsIcon, ImTable as TableIcon } from "react-icons/im";
 import { trpc } from "../utils/trpc";
 
-const iconSize = 8;
-
 export const TopRow = () => {
-  const roundlist = trpc.rounds.roundlist.useQuery().data?.data ?? {
+  const { requestedRound, requestRound, display, setDisplay } =
+    useGlobalContext();
+  const cardLink = "/round/" + requestedRound + "/cards";
+  const tableLink = "/round/" + requestedRound + "/table";
+  const router = useRouter();
+
+  const roundList = trpc.rounds.roundlist.useQuery(undefined, {
+    refetchOnWindowFocus: false,
+  }).data?.data ?? {
     rounds: [],
     latest: 0,
   };
+
+  const iconProps = {
+    size: "1.6rem",
+    marginRight: "1rem",
+  };
+
+  const handleChange = (e: any) => {
+    console.log(e.target.value);
+    requestRound(e.target.value);
+    const href = "/round/" + e.target.value;
+    router.push(href);
+  };
+
   return (
     <>
-      <HStack justify="flex-end" p={4}>
-        <Box flex="1">
-          <Text fontSize="1xl">Beets Wars V2</Text>
-        </Box>
+      <HStack p={4} justifyContent="flex-end">
         <Box>
-          <p>latest: {roundlist.latest}</p>
-          <p>{roundlist.rounds.toString()}</p>
-        </Box>
-        <HStack mt={20} spacing={4}>
-          <Link href="/chart">
-            <Icon as={StatsIcon} h={iconSize} w={iconSize} />
+          <Link href="/bribeform">
+            <Text fontSize="1xl">bribeform</Text>
           </Link>
-          <Link href="/">
-            <Icon as={CardIcon} h={iconSize} w={iconSize} />
-          </Link>
-          <Link href="/">
-            <Icon as={TableIcon} h={iconSize} w={iconSize} />
-          </Link>
-          <CustomConnectButton />
-          <ColorModeSwitcher />
-        </HStack>
+        </Box>
+        <Box style={{ marginRight: "9px" }}>
```

Comment:
this can be <Box mr={2}>, (which will be 8px (I think), but if you really want 9px then mr="9px")